code/datums/emergency_calls/colonists.dm

```diff
@@ -4,7 +4,8 @@
 
 
 /datum/emergency_call/colonist/print_backstory(mob/living/carbon/human/H)
-	to_chat(H, "<B>You are a colonist!</b>")
+	to_chat(H, "<B>You are a simple colonist hoping to be something else.</b>")
```

Comment:
This is from another PR, you'll have to fix it.

Reply:
Aw great... Reverted now.